tests/Makefile.am

```diff
@@ -33,7 +34,8 @@ src_liblogger_la_CFLAGS = $(BUILD_FLAGS)
 # TESTS
 ################################################
 TESTING_FLAGS = -I system_config -I ../src -I include \
-                $(WARNING_FLAGS) $(CPPUNIT_CFLAGS)
+                -std=c++11 \
```

Comment:
Some people are still on 4.4 :). More curious what features of the latest version you're using?